docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Migrating ASP.NET MVC Applications to Windows Containers
+description: Learn how to take an existing ASP.NET MVC application and run it in a Windows Docker Container
+keywords: .NET, .NET Server, .NET Desktop
+author: BillWagner
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-framework-4.6
+ms.technology: dotnet-mvc
+ms.devlang: dotnet
+ms.assetid: c9f1d52c-b4bd-4b5d-b7f9-8f9ceaf778c4
+---
+
+# Migrating ASP.NET MVC Applications to Windows Containers
+
+Running an existing .NET Framework-based application in a Windows container
+requires creating the Docker image that contains your application, and
+starting one or more containers to run that image. This topic explains
+the tasks you must perform to take an existing ASP.NET MVC application
+and deploy it in a Windows container.
+
+You'll start with an existing ASP.NET MVC application. You'll see how to
+build the assets to publish using Visual Studio. You'll use Docker
+to build the image that contains your application, and
+runs that application when it is started. You'll see how to connect to your
+application running in a Windows container.
+
+If you are unfamiliar with Docker, you can learn more about the Docker
+architecture by reading the 
+[Docker Overview](https://docs.docker.com/engine/understanding-docker/)
+on the Docker site. 
+
+The application you'll run in a container is a simple website that
+answers questions randomly. This application
+is a basic MVC application with no authentication support, or database
+storage. That lets you focus on moving the web tier to a container. Later
+topics show how to move and manage persistent storage in containerized
+applications.
```

Comment:
> The application you'll run in a container is a simple website that answers questions randomly. This application is a basic MVC application with no authentication support, or database storage. That lets you focus on moving the web tier to a container. Later topics show how to move and manage persistent storage in containerized applications.

How about:

The application you'll run in a container is a simple website that answers questions randomly. This application is a basic MVC application with no authentication support, or database storage, letting you focus on moving the web tier to a container. Future topics will show how to move and manage persistent storage in containerized applications.